timemachine/md/builders.py

```diff
@@ -46,8 +111,14 @@ def build_protein_system(host_pdbfile: Union[app.PDBFile, str], protein_ff: str,
     solvated_host_coords = strip_units(modeller.positions)
 
     nha = host_coords.shape[0]
+    if mols is not None:
+        water_idxs = np.arange(nha, solvated_host_coords.shape[0])
```

Comment:
nit: present before this PR, but seems worth replacing `nha` with a more readable name

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1339/commits/920122027a067705dacb966f7e50808bd83b6b1c